pkg/readers/dir_reader.go

```diff
@@ -54,6 +53,8 @@ func (p *directoryReader) ApplicationName() (string, error) {
 func (p *directoryReader) EnumManifests(handler func(*models.PackageManifest,
 	PackageReader) error,
 ) error {
+	exclusionMatcher := newPathExclusionMatcher(p.config.Exclusions)
```

Comment:
nitpick: For a future cleanup, we can move initializing `exclusionMatcher` in the `NewXXX` function only because this is pretty much initialization.